### PR TITLE
Structure and Styling for Favorite Ideas Section In Mobile

### PR DESCRIPTION
**Pull request by jaypeasee**
#### What’s this PR do? 
Builds out the structure and styling of the section that holds the favorite idea cards in a mobile layout.

#### Where should the reviewer start?
1. Line 18-101 of `index.html`. Most of this code is duplicated 5 times over to create 5 cards.
2. Line 63-133 of `styles.css`.

#### How should this be manually tested?
1. Compare the colors and sizes to the provided comp in mobile.
2. Review the section in every single device view within the developer tools to make sure the structure stays consistent throughout all of those views.
3. Stretch the page out to about 500 pixels to make sure the content stays appropriately contained.

#### Any background context you want to provide? 
This is the first PR submitted while our group is "dividing and conquering". It is just mobile. Once we complete all sections in mobile, we'll move on to larger breakpoints for tablet and desktop.

#### What are the relevant tickets? 
1. Brigette will have a ticket for the nav section and filter section.
2. Blake will have a ticket for the idea input section.
3. Next steps will to divide and conquer for all sections in larger breakpoints.

#### Screenshots (if appropriate): 
Screenshot of idea cards in mobile

### CODE REVIEW

**Comment by BrigetteDoelp** [comment]
These cards look awesome! Glad to have learned some more basics with grid, as well are get a deeper understanding as to how flexbox works. Let's see about nailing down a workflow with this divide and conquer method.

File: index.html [hunk 1]
```diff
@@ -56,6 +56,48 @@ <h2>Idea Title</h2>
...
+          </div>
+          <div>
+            <h2>Idea Title</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum suscipit consequat nibh, ac interdum dui ultricies id. Integer sit amet elementum felis. Sed pulvinar, risus in luctus aliquet, quam ex sagittis erat, sollicitudin egestas neque dui vel massa.</p>
+          </div>
+          <div class="box-footer">
+            <img src="assets/comment.svg" alt="Add Comment Icon">
+            <h3>Comment</h3>
+          </div>
+        </div>
```
Comment (BlakeDonn) [inline index.html]: This makes sense to copy/paste this so we can test functionality on desktop next.  Let's look into text limits for the `<p>` on 94, want to make sure we can deal with overflow when refactoring.

File: styles.css [hunk 2]
```diff
@@ -60,6 +60,68 @@ img {
 
 /*=== main section ===*/
 
+.idea-cards {
```
Comment (BrigetteDoelp) [inline styles.css:63]: Nice grid work! Really glad to see this in action as I have little experience with grid.

File: styles.css [hunk 3]
```diff
@@ -96,3 +96,16 @@ img {
...
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 2%;
+  width: 90%;
+}
+
+.idea-box {
+  width: 100%;
+  background-color: #FFFFFF;
+}
```
Comment (BlakeDonn) [inline styles.css]: This looks good, let's definitely keep using fr as much as we can for columns in grid. For better scalability.

File: styles.css [hunk 4]
```diff
@@ -136,9 +136,10 @@ img {
...
   background-color: #5356A4;
   height: 40px;
+  display: flex;
+  align-items: center;
+  padding: 0px 20px;
 }
 .box-footer h3 {
   color: #FFFFFF;
-  padding: 0px 20px;
-  display: flex;
```
Comment (BlakeDonn) [inline styles.css]: Do we want to use pixels for this padding because it is so small? Won't affect the Desktop layout I assume?

**Comment by BlakeDonn** [comment]
High level overview: Looks good to merge on my end, definitely matches the example on the project for the mobile view. Some slight things we can refactor in the future but it looks great so far.

**Pull request merged**